include/mbedtls/cipher.h

```diff
@@ -853,6 +851,129 @@ int mbedtls_cipher_crypt( mbedtls_cipher_context_t *ctx,
                   const unsigned char *input, size_t ilen,
                   unsigned char *output, size_t *olen );
 
+#if defined(MBEDTLS_CIPHER_MODE_AEAD)
```

Comment:
Again, this re-introduces a pair of deprecated functions that were removed in #4445 so I assume this was not intentional (probably a mistake when resolving a conflict).